### PR TITLE
feat(base) - handleParamString2, handleParamInteger2

### PR DESCRIPTION
**Pull request by ttodua**

### CODE REVIEW

**Comment by ttodua** [comment]
@carlosmiei can you make a quick approval for this?

File: ts/src/base/Exchange.ts [hunk 1]
```diff
@@ -3957,6 +3957,14 @@ export default class Exchange {
         return [ value, params ];
     }
 
+    handleParamString2 (params: object, paramName1: string, paramName2: string, defaultValue: Str = undefined): [string, object] {
```
Comment (carlosmiei) [inline ts/src/base/Exchange.ts:3960]: first we can't have this return type and second, this is not consistent with the `handleOptionAndParams2` (here the 2 means 2 different params, whereas there it represents 2 different methods), should we update the other implementation?

Comment (ttodua) [reply ts/src/base/Exchange.ts:3960]: return type - can you plz tell wdym about return type? the regular `handleParamString` (without 2) has that same return type

Comment (ttodua) [reply ts/src/base/Exchange.ts:3960]: i think that using `2` in the end of the methods across mostly means two have to provide two keys of the provided dict, but `handleOptionAndParams2` accepts two methods, yeah. as you decide, imo `handleParam2` doesn't include any method-specific parts, so `2` there might be intuitive that it accepts two keys from `params` dict. we might not touch `handleOptionAndParams2` and leave it as is. 
as you tell me

Comment (ttodua) [reply ts/src/base/Exchange.ts:3960]: @carlosmiei can you comment on this ^

**Comment by ttodua** [comment]
@carlosmiei can you give me a hand here? 
```
[ value, params ] = this.handleOptionAndParams (...)
if (condition) {
    [ value, params ] = this.handleOptionAndParams (...)
```
is transpiled into c# like:
```
var valueparametersVariable = this.handleOptionAndParams(...);
value = ((IList<object>)valueparametersVariable)[0];
parameters = ((IList<object>)valueparametersVariable)[1];

if (...) {
   var valueparametersVariable = this.handleOptionAndParams(...);
```

so as you see, `var` is declared two times, in lower scope too, causing errors in build: https://app.travis-ci.com/github/ccxt/ccxt/builds/270262306#L2672

**Comment by ttodua** [comment]
@carlosmiei this should be ready after travis finishes

**Pull request merged**